src/darwin/Framework/CHIP/templates/CHIPClustersObjc-src.zapt

```diff
@@ -31,8 +31,22 @@ using namespace chip::app::Clusters;
 {{#*inline "callbackName"}}{{#if hasSpecificResponse}}{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase responseName}}{{else}}CommandSuccess{{/if}}{{/inline}}
 - (void){{asLowerCamelCase name}}With{{#if (hasArguments)}}Params:(CHIP{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase name}}Params * {{#unless (commandHasRequiredField .)}}_Nullable{{/unless}})params completionHandler{{else}}CompletionHandler{{/if}}:({{>command_completion_type command=.}})completionHandler
 {
+  CHIPCommandArguments * arguments;
+  {{#if mustUseTimedInvoke}}
+  arguments = [CHIPCommandArguments new];
+  arguments.timedInvokeTimeoutMs = @10000;
+  {{/if}}
+  [self {{asLowerCamelCase name}}WithArguments: arguments{{#if (hasArguments)}} params:params completionHandler{{else}} completionHandler{{/if}}:completionHandler];
+}
+- (void){{asLowerCamelCase name}}WithArguments: (CHIPCommandArguments * _Nullable)arguments{{#if (hasArguments)}} params:(CHIP{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase name}}Params * {{#unless (commandHasRequiredField .)}}_Nullable{{/unless}})params completionHandler{{else}} completionHandler{{/if}}:({{>command_completion_type command=.}})completionHandler
+{
+    chip::Optional<uint16_t> invokeTimeOutMs;
```

Comment:
This is not a timeout for the invoke.  It's a timeout for the timed part of things...

Reply:
Updated. Thank you.